<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="c1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="profList"/>
      </item>
      <item>
       <widget class="QWidget" name="c1_Buttons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="profNew">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profDelet">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="c2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="backend">
         <attribute name="title">
          <string>Backend</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTabWidget" name="backendTabs">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="general">
             <attribute name="title">
              <string>General</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>XDD Directory</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QWidget" name="widget_3" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLineEdit" name="G_XDDDir"/>
                    </item>
                    <item>
                     <widget class="QToolButton" name="G_XDDDirSelct">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="sm">
             <attribute name="title">
              <string>Snapshot Manager</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Snapshot Interval</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="SM_interval"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>448</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="IH">
             <attribute name="title">
              <string>Input Handler</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Cleanup interval</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>EPL Wireshark Frame Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Loop wait timeout</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="spinBox"/>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSpinBox" name="spinBox_4"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="spinBox_2"/>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Delete Cycles after</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Check prefetch interval</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Prefetch size</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item row="4" column="1">
                  <widget class="QSpinBox" name="spinBox_5"/>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSpinBox" name="spinBox_3"/>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Cycle</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Cycle</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Cycle</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>ms</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>ms</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>280</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="nodes">
         <attribute name="title">
          <string>Nodes</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QListWidget" name="nodesList">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QPushButton" name="nodeAdd">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="nodeRM">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QWidget" name="widget_9" native="true">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="1">
                 <widget class="QCheckBox" name="N_autoDetect">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Autodetect Node type</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Base Profile File</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Special Profile File</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="N_Base"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="N_Special"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>441</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="python">
         <attribute name="title">
          <string>Python</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="apply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nodeAdd</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>newNode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeRM</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>deleteNode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profNew</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>newProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profDelet</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>deleteProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>profChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodesList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>SettingsWindow</receiver>
   <slot>nodeChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>apply()</slot>
  <slot>reset()</slot>
  <slot>newProfile()</slot>
  <slot>deleteProfile()</slot>
  <slot>newNode()</slot>
  <slot>deleteNode()</slot>
  <slot>profChange()</slot>
  <slot>nodeChange()</slot>
 </slots>
</ui>
